index.html

```diff
@@ -1044,48 +1060,101 @@
             <li>Return <var>P</var>, but continue running these steps in
             parallel.
             </li>
-            <li>For each <var>known connection</var> in the <a>set of
-            presentations</a>,
-              <div style="margin-left: 2em">
-                If <a>presentation connection state</a> of <var>known
-                presentation</var> is not <a for=
-                "PresentationConnectionState">terminated</a>, the
-                <a>presentation URL</a> of <var>known connection</var> is equal
-                to the <a>presentation request URL</a> of
-                <var>presentationRequest</var>, and the <a>presentation
-                identifier</a> of <var>known connection</var> is equal to
-                <var>presentationId</var>, run the following steps:
-                <ol>
-                  <li>Let <var>S</var> be the <a>presentation connection</a> of
-                  <var>known connection</var>.
-                  </li>
-                  <li>
-                    <a>Resolve</a> <var>P</var> with <var>S</var>.
-                  </li>
-                  <li>
-                    <a>Queue a task</a> to <a>fire</a> a <a>trusted event</a>
-                    with the name <a>connectionavailable</a>, that uses the
-                    <a>PresentationConnectionAvailableEvent</a> interface, with
-                    the <a for=
-                    "PresentationConnectionAvailableEvent">connection</a>
-                    attribute initialized to <var>S</var>, at
-                    <var>presentationRequest</var>. The event must not bubble,
-                    must not be cancelable, and has no default action.
-                  </li>
-                  <li>
-                    <a>Establish a presentation connection</a> with
-                    <var>S</var>.
-                  </li>
-                  <li>Abort all remaining steps.
-                  </li>
-                </ol>
-              </div>
+            <li>Search the <a>set of controlling presentations</a> for a
+            <a>PresentationConnection</a> that meets the following criteria:
+            its <a>controlling browsing context</a> is the current <a>browsing
```

Comment:
Hmm, the second "Search" in that procedure is meant to address that case.

My reasoning is the following:

- if there is a `PresentationConnection` for the same ID and URL associated with the current browsing context, we should reuse it.
- if not, we cannot reuse a `PresentationConnection` attached to another browsing context (it's only useful to check that the ID and URL are correct), and we need to create a new `PresentationConnection`. The only difference with the `start()` case is that the controlling user agent would somehow pass the presentation identifier to the receiving user agent to connect to the right receiving browsing context.